models/transaction.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * This file contains wrapper functions to interact with the DB.
+ * This will contain the DB schema if we start consuming an ORM for managing the DB operations
+ */
+
+const firestore = require('../utils/firestore')
+const transactionsModel = firestore.collection('transaction')
+
+/**
+ * fetch latest 10 transactions from DB from specific userId provided
```

Comment:
Please have a look at line 10, needs to be changed. I meant the number of transactions

Reply:
Changed to  more generic description